crates/router/src/connector/airwallex.rs

```diff
@@ -91,6 +92,28 @@ impl ConnectorCommon for Airwallex {
     }
 }
 
+impl ConnectorValidation for Airwallex {
+    fn validate_capture_method(
+        &self,
+        capture_method: enums::CaptureMethod,
+    ) -> CustomResult<(), errors::ConnectorError> {
+        let unsupported_capture_method = match capture_method {
+            enums::CaptureMethod::Automatic | enums::CaptureMethod::Manual => None,
+            enums::CaptureMethod::ManualMultiple => Some("manual_multiple"),
+            enums::CaptureMethod::Scheduled => Some("schedule"),
+        };
+        if let Some(capture_method) = unsupported_capture_method {
+            Err(errors::ConnectorError::NotSupported {
```

Comment:
Move the error construction logic of NotSupported and NotImplemented inside crates/router/src/connector/utils.rs and reuse the code